_data/simple-icons.json

```diff
@@ -2710,6 +2710,11 @@
             "hex": "003399",
             "source": "https://global.epson.com/IR/library/"
         },
+        {
+            "title": "Equinix Metal",
+            "hex": "D10810",
```

Comment:
Where did you get this color from?

Reply:
From their Website out of the CSS

Reply:
Could you be more specific? I tried the color of [the FAB](https://material.io/components/buttons-floating-action-button) on the homepage (`.intercom-lightweight-app-launcher.intercom-launcher`) and it has`#ED1C24`...